ElectronNET.API/BridgeConnector.cs

```diff
@@ -195,7 +332,7 @@ public static async Task<T> OnResult<T>(string triggerEvent, string completedEve
                         EventTasks<T>.DoneWith(completedEvent, eventKey, taskCompletionSource);
                     });
 
-                    Emit(triggerEvent, args);
+                    await EmitAsync(triggerEvent, args);
```

Comment:
`await EmitAsync(triggerEvent, args).ConfigureAwait(false);`

every async should call `ConfigureAwait(false);` where possible in the method chains.